uploaded code

kkamenskayaa opened the pull request.

kkamenskayaa committed:
uploaded code

mkouznetsov-git commented on src/test/java/Praktikum/OrderTest.java:
```diff
@@ -0,0 +1,89 @@
+
+import PageObjects.MainPage;
+import PageObjects.OrderPage;
+import org.junit.After;
```
Много лишних импортов -- удаляй

mkouznetsov-git commented on src/test/java/Praktikum/OrderTest.java:
```diff
@@ -0,0 +1,89 @@
+
+
+@RunWith(Parameterized.class)
+public class OrderTest {
```
По заданию нужно проверить обе кнопки заказа. Здесь пока только тест на первую кнопку. Надо добавить вторую.

mkouznetsov-git commented on src/test/java/Praktikum/OrderTest.java:
```diff
@@ -0,0 +1,89 @@
+        orderPage.fillName(name);
+        orderPage.fillSurname(surname);
+        orderPage.fillAddress(address);
+        orderPage.selectMetro(metro);
```
Здесь тест падает с ошибкой:
org.openqa.selenium.ElementClickInterceptedException: Element <div class="Order_Text__2broi"> is not clickable at point (332,699) because another element <div class="App_CookieText__1sbqp"> obscures it

kkamenskayaa committed:
Исправления после ревью

1 Убраны неиспользуемые импорты
2 TestOrder разделен на два теста: TestOrderHeader() и TestOrderBody()
3 Исправлено падение тестов заказа: Добавлены проверка исчезания флоатера с куками и ожидание загрузки хедера окна заказа
